src/vs/workbench/contrib/webview/browser/overlayWebview.ts

```diff
@@ -144,6 +147,17 @@ export class OverlayWebview extends Disposable implements IOverlayWebview {
 	}
 
 	public layoutWebviewOverElement(element: HTMLElement, dimension?: Dimension, clippingContainer?: HTMLElement) {
+		this.doLayoutWebviewOverElement(element.dimension, clippingContainer);
```

Comment:
I think you meant to say `element, dimension` here

Reply:
yes, sorry, updating and pushing